lib/scrabble.rb

```diff
@@ -1,9 +1,33 @@
 class Scrabble
 
-  def score(word)
-    1
+
+
+  def score(word = nil)
+    score = 0
+    if word != nil
+      individual_letters = word.split('')
+      individual_letters.each do |letter|
```

Comment:
I'd also love to see you implementing more advanced enums; this could be a good case for map as you're 'mapping' scores onto each letter (then you can sum at the end). There are many possible solutions though & glad you got it to work!